numpy/core/tests/test_multiarray.py

```diff
@@ -7254,9 +7254,31 @@ def test_view_assign(self):
         assert_equal(arr, -100)
         # after resolve, the two arrays no longer reference each other
         assert_(not arr_wb.ctypes.data == 0)
+        assert_equal(arr_wb.base, None)
```

Comment:
Just curious, but shouldn't the base point to something valid even if the array is about to be destroyed?

Reply:
`base` is meant to be `None` for an array that owns its data, see [documentation for base](https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.ndarray.base.html#numpy.ndarray.base). The point of the test is to show that the two arrays `arr_wb` and `arr` are totally independent after the call to `PyArray_ResolveWritebackIfCopy` or `PyArray_DiscardWritebackIfCopy`.